spec/models/cart_item_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe CartItem, type: :model do
+  let(:cart_item) {create(:cart_item, pricing: pricing, quantity: quantity)}
+  let(:pricing) {}
+  let(:quantity) {10}
+
+  describe '#total_price' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.